app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="container-sm">
+  <div class="row">
+
+    <div class="col-6">
+      <div class="card" style="width: 18rem;">
+        <div class="card-header bg-danger text-white">
+          Pending Friendships
+        </div>
+        <% if @pending_friends %>
```

Comment:
- [ ] Please remove view logic, conditionals and loops, from the view and into helper methods. Read this article about why your views should not have a lot of logic and how to refactor them: [Toptal article.](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make)

Reply:
Resolved